test/extended/build_context_dir_test.go

```diff
@@ -0,0 +1,76 @@
+// +build integration,extended,!no-etcd,!no-docker,docker
+
+package extended
+
+import (
+	"testing"
+	"time"
+
+	kapi "github.com/GoogleCloudPlatform/kubernetes/pkg/api"
+	buildapi "github.com/openshift/origin/pkg/build/api"
+	"github.com/openshift/origin/test/integration"
+)
+
+func init() {
+	integration.RequireDocker()
+	integration.RequireEtcd()
+}
+
+func TestExtendedBuildContextDir(t *testing.T) {
+	integration.DeleteAllEtcdKeys()
+	openshift, _ := integration.StartTestAllInOne()
```

Comment:
yeah, I hate to type integration. everytime I want to call this... also funny fact... we are not using StartServer anywhere in integration....